stryker.conf.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = function(config) {
+  config.set({
+    mutator: 'javascript',
+    packageManager: 'npm',
+    reporters: ['html', 'clear-text', 'progress'],
+    testRunner: 'command',
+    commandRunner: { command: 'npm run unittests' },
+    transpilers: [],
+    coverageAnalysis: 'all',
+    mutate: ['lib/*.js'],
```

Comment:
Is there any way to exclude `lib/speed.js` from here since there are no tests for it?

Reply:
Done by prepending `!` to the filename to ignore.